benchmarks/benchmark/tools/locust-load-inference/locust-docker/locust-tasks/run.sh

```diff
@@ -24,17 +24,20 @@ if [[ "$LOCUST_MODE" = "master" ]]; then
     # For inferencing workloads with large payload having no wait time is unreasonable.
     # This timeout is set to large amount to avoid user tasks being killed too early.
     # TODO: turn timeout into a variable.
-    LOCUS_OPTS="$LOCUS_OPTS --master --stop-timeout 10800"
+    LOCUST_OPTS="$LOCUST_OPTS --master "
+    if [[ "$STOP_TIMEOUT" != 0 ]]; then
+        LOCUST_OPTS="$LOCUST_OPTS --stop-timeout $STOP_TIMEOUT"
```

Comment:
OOC, why does this need to be an opt-in var? Is this related to making the runner work?

Reply:
Addressed in the previous comment